PL_Biotech_PrisonersInteractionMode.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>

```

Comment:
This file is the same as previous one. I believe it was added by mistake, right?

Reply:
NO~Is Biotech content.

Because Im not sure whether it is displayed normally, I copied it from the source code and modified it into a language pack

Reply:
![obraz](https://user-images.githubusercontent.com/4631961/207277340-eb623532-f1da-4f8c-b6dd-03a02d16fe50.png)
Its screenshot from bio tech with only first file. Please verify if its translated correctly.

Reply:
![obraz](https://user-images.githubusercontent.com/4631961/207278064-7817e753-feea-43dd-9d01-2bf658dc835b.png)
![obraz](https://user-images.githubusercontent.com/4631961/207278143-d6f96f39-97b2-4708-8077-a647de7f520c.png)

Reply:
OK~I will edit it